### ISSUE TITLE
Is "Sex" correctly labelled in the Pedigree example and Sex documentation 

### ISSUE THREAD

**Issue by slaurie** [phenopackets/phenopacket-schema#391]
Hi @julesjacobsen 

Is Sex correctly labelled in the Pedigree/Sex schemas?

In genetics/genomics, we usually use "1" for male, and "2" for female (easy to remember as equivalent to the normal number of X chromosomes per sex), but in the documentation and the examples for the Pedigree (https://phenopacket-schema.readthedocs.io/en/latest/pedigree.html) and Sex fields (https://phenopacket-schema.readthedocs.io/en/latest/sex.html#rstsex), it indicates that females should be encoded as 1 and males as 2. 

Unfortunately the NICT links on the Sex page didn't work for me, so I can't check if they do have F1 and M2. However, if this is the case, then there is an inconsistency between NICT and what is defined in PLINK, so a choice needs to be made, as the documentation is inconsistent. Personally, I would suggest we stick with PLINK.

This may seem trivial, but obviously could lead to a lot of miscoding that will be a pain for users to rectify later.

**Comment by slaurie** [comment]
P.S. If we were to stick with PLINK values, this would also affect Karyotipic Sex values
https://phenopacket-schema.readthedocs.io/en/latest/karyotypicsex.html#data-model

**Comment by julesjacobsen** [comment]
Yes, they are correct - note that the ORDINAL value isn't what should be used in the phenopacket, the value in the NAME field for these docs is what should be used, as shown in the example : https://phenopacket-schema.readthedocs.io/en/latest/sex.html#example 

Example code for converting from PED /PLINK to Phenopackets would be

```java
int pedSex = ...
var phenopacketSex = switch(pedSex) {
    case 1 -> FEMALE;
    case 2 -> MALE;
    case 3 -> OTHER_SEX;
    default -> UNKNOWN_SEX;
``` 

It's unfortunate that the ordinals don't match the PLINK values but they are different things.

**Comment by slaurie** [comment]
Thanks for confirming, that your encoding is correct, and I noticed that the actual terms MALE and FEMALE are used explicitly which is great.

However, there is still an error on the "Pedigree" documentation (https://phenopacket-schema.readthedocs.io/en/latest/pedigree.html),
 where it states:

"In a PED file, the sex of individuals is encoded as a “1” for females, “2” for males, and “0” for unknown. Phenopackets uses [Sex](https://phenopacket-schema.readthedocs.io/en/latest/sex.html#rstsex) instead.",
 
and this is also the encoding you have used in the examples., whereas the PLINK PED documentation (https://zzz.bwh.harvard.edu/plink/data.shtml) has

"Sex (1=male; 2=female; other=unknown)"

So it might be worth correcting this somehow. I do much prefer the explicit MALE and FEMALE.

Thanks, Steve

**Comment by pnrobinson** [comment]
@slaurie thanks for this. 
I am update the documentation as follows:
```bash

In a PED file, affected persons are encoded with "2", and unaffecteds by "1"
(a "0" is used if no information is available). Instead, Phenopackets uses an enumeration as shown in the table.

In a PED file, the sex of individuals is encoded as a "2" for females, "1" for males, and "0" for unknown. Phenopackets
uses :ref:`rstsex` instead (Note that the enumeration used for Sex by the Phenoapcket Schema has 1 for FEMALE and 2 for MALE, but this is internal and is not visible in the JSON-serialized Phenopacket).
```

**Comment by psnairne** [comment]
Hi, I was reading this part of the documentation and got confused, I think it might need to be changed again? Two things:

1. 

> In a PED file, affected persons are encoded with “2”, and unaffecteds by “1” (a “0” is used if no information is available). Instead, Phenopackets uses an enumeration as shown in the table.

The 

> instead

 doesn't make sense here, they are the same enumeration

2.
```
 "family 1"    "kindred 1A"    "FATHER"    "MOTHER"    2   2
"family 1"    "kindred 1B"    "FATHER"    "MOTHER"    2   2
"family 1"    "MOTHER"      0   0   1   1
"family 1"    "FATHER"      0   0   2   1
```

> In a PED file, affected persons are encoded with “2”, and unaffecteds by “1”


so the PED lines above are incorrect, it should be 

```
 "family 1"    "kindred 1A"    "FATHER"    "MOTHER"    2   1
"family 1"    "kindred 1B"    "FATHER"    "MOTHER"    2   1
"family 1"    "MOTHER"      0   0   1   2
"family 1"    "FATHER"      0   0   2   2
```

**Comment by slaurie** [comment]
@psnairne 
I think the original PEDigree example is correct (i.e. the top one of your pair). I read it as two male (2) children (since 2 is male in phenopackets schema), both affected (2), and then their mother (1, female) and father (2), who are both unaffected (1).

This is in line with the documentation.

**Comment by psnairne** [comment]
I was confused because it says above the example 

`Here we show a pedigree in PED format`, followed by later `In a PED file, the sex of individuals is encoded as a “2” for females...` 

Though I understand the point is that the example is in PED format with the Phenopacket encoding. 

Perhaps the explanation for the different Sex encodings could be moved upwards (currently it is in the Affected Status section). 

e.g. 

`Here we show a pedigree in PED format, this contains two male siblings which share an abnormal (affected) phenotype and their two normal (unaffected) parents.`

->

`Here we show a pedigree in PED format, this contains two male siblings which share an abnormal (affected) phenotype and their two normal (unaffected) parents. In a typical PED file, the sex of individuals is encoded as a “2” for females, “1” for males, and “0” for unknown. We use here the Phenopacket encoding for [Sex](https://phenopacket-schema.readthedocs.io/en/latest/sex.html#rstsex) instead (Note that the enumeration used for Sex by the Phenopacket Schema has 1 for FEMALE and 2 for MALE, but this is internal and is not visible in the JSON-serialized Phenopacket).`
